### PR TITLE
docs: add posts related to server actions

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Add posts describing the steps to run actions to get status, start, stop, restart and remove servers

## Issues resolved by this PR

- [x] [DVN-10755]
- [x] [DVN-10756]
- [x] [DVN-10757]
- [x] [DVN-10758]
- [x] [DVN-10759]
- [x] [DVN-10760]

## More info

"Get a Server's Status" post preview
get-server-status

"Remove a Server" post preview
remove-server

"Stop a Server" post preview
stop-server

"Start a Server" post preview
start-server

"Restart a Server" post preview
restart-server

[DVN-10755]: https://devopness.atlassian.net/browse/DVN-10755?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10756]: https://devopness.atlassian.net/browse/DVN-10756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10757]: https://devopness.atlassian.net/browse/DVN-10757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10758]: https://devopness.atlassian.net/browse/DVN-10758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10759]: https://devopness.atlassian.net/browse/DVN-10759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10760]: https://devopness.atlassian.net/browse/DVN-10760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/servers/stop-server.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+draft: true
+title: Stop a Server
+intro: Sometimes you may want to interrupt a server to stop a long running task. Stop a server to force any running tasks to end.
```
Comment (Silva97) [inline docs/servers/stop-server.md:4]: > Sometimes you may want to interrupt a server to stop a long running task. Stop a server to force any running tasks to end.

Essa descrição não deixa muito claro o efeito colateral de um `stop`. Pois um usuário mais leigo nesse assunto pode não entender que "stop a server" significa interromper completamente o servidor, o que pode implicar em consequências indesejadas como, por exemplo, a mudança do IP público do servidor ao iniciá-lo novamente (o que implica na necessidade de atualizar domínios posteriormente, uma ação feita por fora do Devopness).

É que lendo da forma que tá escrito me parece que o único efeito colateral do `stop` é interromper as actions em execução no servidor, ou algo nessa linha.

Sei que a ideia é ser um step-by-step e evitar "textão", mas como estamos falando de algo que pode derrubar um sistema em produção e criar alguma dor de cabeça para o usuário, talvez exija um pouco mais de detalhes técnicos aqui.

O que acha?

Comment (thlmenezes) [reply docs/servers/stop-server.md:4]: I wholeheartedly agree with you,

I myself didn't quite understand the full impact this action could have in the workflow; we could add an introduction with notes like those proposed on #686 in the `docs/environments/archive-environment.md` file.

Could you help me with what warnings should we give to our users to ponder about before stopping a server, besides the public IP change?

Comment (Silva97) [reply docs/servers/stop-server.md:4]: Sure. I think to warn the user that "stop" the server will temporary make the server out of work and it's impact on:

1. Any applications on this server will stop of working until the server is stopped
2. Daemons will be stopped too
3. If the server don't have a elastic IP configured to it (Devopness don't configure this today, the user need to configure it on the cloud provider), the public IP address of the server will be updated. So the user should update any domain pointing for this server.

Comment (thlmenezes) [reply docs/servers/stop-server.md:4]: @Silva97 does these notes include all the necessary warnings?

> Note #1: Any applications or daemons on this server will stop working as a consequence of the server being stopped.
> Note #2: Your server's public IP address will be updated the next time the server is started, unless you have configured an elastic IP on your cloud provider management panel; meaning, the next time this server is started, you will need to update any domains pointing to this server.

Comment (Silva97) [reply docs/servers/stop-server.md:4]: Yes.

**Review by luizvi — APPROVED** [review]

**Review by luizvi — APPROVED** [review]

**Review by jfoliveira — APPROVED** [review]
LGTM

**Pull request merged**